packages/udaru-hapi-server/.labrc.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+    coverage: false,
```

Comment:
why is this set to false?

Reply:
The lab-config file is new here. And since coverage was not yet created *at all* for this sub-package, it would result in coverage 0% and failed test by adding `coverage: true` check.
Setting it to *false* in this case does not make a difference. Coverage generation should be added with another issue I suggest.